### PR TITLE
BUGFIX: TMA-1121 use current user call instead of expensive bootstrap

### PR DESCRIPTION
**Pull request by panjan**

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/738/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1561/ : FAILURE in 1m 42s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1822/ : FAILURE in 1m 17s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-check-pipeline/54/ : SUCCESS in 5m 44s

File: lib/gooddata/rest/connection.rb [hunk 1]
```diff
@@ -175,13 +175,13 @@ def connect(username, password, options = {})
           merge_headers!(headers)
           get('/gdc/account/token', @request_params)
 
-          @user = get(get('/gdc/app/account/bootstrap')['bootstrapResource']['accountSetting']['links']['self'])
+          @user = get('/gdc/account/profile/current')
```
Comment (yenkins) [inline lib/gooddata/rest/connection.rb]: info: Calls 'get('/gdc/account/profile/current')' 2 times (DuplicateMethodCall) (reek)

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/739/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1562/ : SUCCESS in 2m 36s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1823/ : FAILURE in 1m 02s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-check-pipeline/55/ : SUCCESS in 5m 54s

**Comment by panjan** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/777/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1598/ : SUCCESS in 1m 14s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1859/ : SUCCESS in 1m 10s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/38/ : SUCCESS in 9m 08s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/780/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1600/ : SUCCESS in 1m 08s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1861/ : SUCCESS in 1m 12s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/42/ : FAILURE in 1m 22s

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/782/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1549/ : FAILURE in 11s
- https://checklist.intgdc.com/job/lcm/job/gooddata-ruby-lcm-integration-tests-zuul/109/ : FAILURE in 34m 02s

**Pull request closed without merging**